### PR TITLE
added labs for create_dataset.ipynb and a_features.ipynb plus lab maintenance

### PR DESCRIPTION
**Pull request by vijaykyr**
changes to a_feature.ipynb:
Add np.random.seed(seed=1) to make results reproducible
Added tensorboard
Fixed num_epochs in train_input fn to be None instead of 8
Reduce training from 5000 to 2000 steps to prevent overfitting
Added more explanation around engineered features and hints to lab
Added section to inspect weights

### CODE REVIEW

**Comment by lakshmanok** [comment]
Nice!
